src/engraving/libmscore/tie.cpp

```diff
@@ -877,6 +964,7 @@ TieSegment* Tie::layoutBack(System* system)
 
     segment->layoutSegment(PointF(x, sPos.p2.y()), sPos.p2);
     segment->setSpannerSegmentType(SpannerSegmentType::END);
+    segment->adjustX();
```

Comment:
This line indeed fixes the issue with ties over system breaks

Reply:
yep, easy fix. though it seems like we might need a rebase now because the build is failing, hold

Reply:
Yep, my PR #9299 should fix this